docs/mobile-apps/features/audio-capture.md

```diff
@@ -0,0 +1,81 @@
+---
+id: audio-capture
+title: Audio Capture
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Audio Capture 
```

Comment:
@aoleszyn Per our convo, please remove this (no headers needed for introductory paragraph).

Reply:
Header removed